js/script.js

```diff
@@ -1,3 +1,9 @@
+import {Popup} from './Popup.js';
+import {Userinfo} from './Userinfo.js';
+import {FormValidator} from './FormValidator.js';
+import {Card} from './Card.js';
+import {CardList} from './CardList.js';
+import {Api} from './api.js';
```

Comment:
Нужно исправить: в режиме разработки происходит подключение к API по протоколу `HTTPS`.
В режиме разработки должно быть по протоколу `HTTP`. Потому что режим разработки сам запускается на этом протоколе.

Reply:
```suggestion
const api = new Api({
        baseUrl: NODE_ENV==='development' ? 'http://praktikum.tk/cohort8' : 'https://praktikum.tk/cohort8',
```
Изменил baseUrl в js/script.js